core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -440,6 +463,119 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
+                  ng-click="onClickThread(thread.exploration_id, thread.thread_id, thread.exploration_title)"
+                  class="oppia-feedback-tab-row protractor-test-feedback-thread">
+                <td class="col-lg-1 col-md-1 col-sm-1">
+                  <span ng-class="getLabelClass(thread.status)"><[getHumanReadableStatus(thread.status)]></span>
+                </td>
+                <td class="col-lg-3 col-md-3 col-sm-3" style="width: 21%;">
+                  <span ng-if="thread.author_second_last_message">
+                    <span ng-if="thread.second_last_message_read"><[thread.author_second_last_message | truncate:7]>, </span>
+                    <span ng-if="!thread.second_last_message_read">
+                      <strong><[thread.author_second_last_message | truncate:7]>, </strong>
+                    </span>
+                  </span>
+                  <span ng-if="thread.last_message_read"><[thread.author_last_message | truncate:7]></span>
+                  <span ng-if="!thread.last_message_read">
+                    <strong><[thread.author_last_message | truncate:7]></strong>
+                  </span>
+                  <span class="feedback-thread-message-count">(<[thread.total_no_of_messages]>)</span>
+                </td>
+                <td class="col-lg-6 col-md-6 col-sm-6" style="width: 58%;">
+                  <span ng-if="thread.total_no_of_messages > 1 && thread.second_last_message_read && thread.last_message_read"><[thread.exploration_title]> -</span>
+                  <span class="protractor-test-feedback-exploration" ng-if="thread.total_no_of_messages === 1 && thread.last_message_read"><[thread.exploration_title]> -</span>
+                  <span ng-if="thread.total_no_of_messages === 1 && !thread.last_message_read">
+                    <strong><[thread.exploration_title]> -</strong>
+                  </span>
+                  <span ng-if="thread.total_no_of_messages > 1 && (!thread.last_message_read || !thread.second_last_message_read)">
+                    <strong><[thread.exploration_title]> -</strong>
+                  </span>
+                  <span ng-if="thread.last_message_text"><[thread.last_message_text | truncate:50]></span>
+                  <span ng-if="!thread.last_message_text"><i>Suggestion</i></span>
+                <td class="col-lg-2 col-md-2 col-sm-2">
+                  <span><[getLocaleAbbreviatedDatetimeString(thread.last_updated)]></span>
+                </td>
+              </tr>
+            </table>
+
+            <table class="table" ng-if="feedbackThreadActive">
+              <tr>
+                <td>
+                  <button ng-click="backFromThread()"
+                          class="btn btn-default"
+                          style="padding: 0 3px;">
+                    <i class="material-icons oppia-vcenter" title="Return to list of messages"></i>
```

Comment:
Not sure why there's an unreadable character here. Shouldn't you use a code like `&#x<[something]>;`, following the [Material Icons site](https://material.io/icons/)?